nkms/blockchain/eth/chains.py

```diff
@@ -77,8 +79,61 @@ def wait_for_receipt(self, txhash, timeout=None) -> None:
         result = self._chain.wait.for_receipt(txhash, timeout=timeout)
         return result
 
-# class TestRpcBlockchain:
+
+class TesterBlockchain(TheBlockchain):
+    """Transient, in-memory, local, private chain"""
+
+    _network = 'tester'
+
+    def wait_time(self, wait_hours, step=50):
+        """Wait the specified number of wait_hours by comparing block timestamps."""
+
+        end_timestamp = self._chain.web3.eth.getBlock(
+            self._chain.web3.eth.blockNumber).timestamp + wait_hours * 60 * 60
+        while self._chain.web3.eth.getBlock(self._chain.web3.eth.blockNumber).timestamp < end_timestamp:
+            self._chain.wait.for_block(self._chain.web3.eth.blockNumber + step)
+
+    def spawn_miners(self, miner_agent, addresses: list, locktime: int, random_amount=False) -> list():
+        """
+        Deposit and lock a random amount of tokens in the miner escrow
+        from each address, "spawning" new Miners.
+        """
+        from nkms.blockchain.eth.actors import Miner
+
+        miners = list()
+        for address in addresses:
+            miner = Miner(miner_agent=miner_agent, address=address)
+            miners.append(miner)
+
+            if random_amount is True:
+                amount = (10 + random.randrange(9000)) * miner_agent._deployer._M
```

Comment:
To be clear for the public: `random.randrange` is _only_ being used to calculate random stake amounts _for testing and testing only_.

Reply:
Let's also have a comment to this effect.

`random.choice('hangs head in shame re: too little entropy', 'forgot to use ChaCha/Salsa for my password generator')`